### DISCUSSION TITLE
Install Homebrew using a .pkg installer on Mac?

### DISCUSSION THREAD

**Discussion by splinnik** [Homebrew/discussions#5509] [Getting started] [answered]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

Hi,

I couldn't find an actual .pkg installer for homebrew in the brew repo, so I made one called BrewInstaller.pkg from brew/package artifacts using pkgbuild, productsign, & notarytool. It got built, signed, & notarized. Yay! However, when I tried to run BrewInstaller.pkg on a different machine, I got "The package is trying to install content to the system volume. Contact the software manufacturer for assistance". Is there a workaround for this so I don't have to ask my non-technical users to install homebrew from the command line?

**Comment by ZhongRuoyu** [comment, marked as answer]
> I couldn't find an actual .pkg installer for homebrew in the brew repo

You can download it from [Homebrew's latest GitHub release](https://github.com/Homebrew/brew/releases/latest).

**Reply by splinnik** [reply]
Thanks!!!!

**Reply by SMillerDev** [reply]
For next time, https://brew.sh has a link there just below the terminal command.

**Comment by splinnik** [comment]
What params were used (not the secret values, of course, just the syntax) for pkgbuild, productsign, & notarytool that enabled the HomeBrew installer to write to system dir?

**Reply by SMillerDev** [reply]
https://github.com/Homebrew/brew/blob/HEAD/.github/workflows/pkg-installer.yml

**Reply by splinnik** [reply]
Nice! How does one run that .yml?

**Reply by SMillerDev** [reply]
It's a GitHub Actions definition so if you have this in a repository on GitHub, it matches the requirements to trigger CI and all secrets are set it will run. You can copy the command to run it locally too.

**Comment by PatrickRoumanoff** [comment]
the pkg is missing for 4.3.21 - how do I report this issue?

**Reply by SMillerDev** [reply]
You can make an issue at https://github.com/homebrew/brew
